build/scripts/build.ps1

```diff
@@ -486,10 +486,18 @@ function Test-XUnit() {
     }
     
     try {
```

Comment:
nice! is exception the one we most interested in?

Reply:
I just had this change in for diagnostics. For now I've pulled it out; the files it creates on build failure are huge.

Reply:
can we make it ignore cancellation exception? I think that is probably reason it got so big since we throws zillion of them.

also, how about zipping the etl file like prerfview at the end? if size is the issue.

Reply:
I'll see about adding this in a future change and/or making it a branch usable by the viper.